component/src/governance/state_key.rs

```diff
@@ -1,4 +1,4 @@
-use penumbra_crypto::IdentityKey;
+use penumbra_crypto::stake::IdentityKey;
```

Comment:
I rearranged the staking-related contents of the `crypto` crate into a `stake` module.  The crate/module organization continues to be a mess.